submissions/jeremy-sung/scripts/article.js

```diff
@@ -3,13 +3,24 @@ var articles = [];
 function Article (opts) {
   // TODO: Use the js object passed in to complete this contructor function:
   // Save ALL the properties of `opts` into `this`.
+  //<--DONE -->
+  this.title = opts.title;
+  this.category = opts.category;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
   this.author = opts.author;
```

Comment:
Very nice getting all the properties. We will discuss a dynamic approach to accessing all properties soon!